lib/ledger_sync/adaptors/netsuite/records/customer.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module LedgerSync
+  module Adaptors
+    module NetSuite
+      module Records
+        class Customer < NetSuite::Record
+          attribute :email, type: LedgerSync::Type::String
+          attribute :companyName, type: LedgerSync::Type::String
```

Comment:
I decided to follow the same convention of the ledger in terms of naming record attributes.  I figured this would be helpful if you were looking at the ledger documentation for attribute help.